cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CassandraClusterInfo.java

```diff
@@ -406,22 +401,15 @@ private TokenRangeMapping<RingInstance> getTokenRangeReplicas()
                                            .map(RingInstance::new)
                                            .collect(Collectors.toSet());
 
-            blockedInstances = response.replicaMetadata()
-                                       .values()
-                                       .stream()
-                                       .map(RingInstance::new)
-                                       .filter(this::instanceIsBlocked)
-                                       .collect(Collectors.toSet());
-
-            Set<String> blockedIps = blockedInstances.stream().map(i -> i.ringInstance().address())
-                                                     .collect(Collectors.toSet());
-
             // Each token range has hosts by DC. We collate them across all ranges into all hosts by DC
             writeReplicasByDC = response.writeReplicas()
                                         .stream()
                                         .flatMap(wr -> wr.replicasByDatacenter().entrySet().stream())
-                                        .collect(Collectors.toMap(Map.Entry::getKey, e -> new HashSet<>(e.getValue()),
-                                                                  (l1, l2) -> filterAndMergeInstances(l1, l2, blockedIps)));
+                                        .collect(Collectors.toMap(Map.Entry::getKey,
+                                                                  e -> new HashSet<>(e.getValue()), (l1, l2) -> {
+                                                                      l1.addAll(l2);
+                                                                      return l1;
+                                                                  }));
```

Comment:
Place the merge function in new line. 

```suggestion
                                    .collect(Collectors.toMap(Map.Entry::getKey,
                                                                  e -> new HashSet<>(e.getValue()),
                                                                  (l1, l2) -> {
                                                                      l1.addAll(l2);
                                                                      return l1;
                                                                  }));
```